tests/PriorityMapTest.php

```diff
@@ -115,7 +125,7 @@ public function testGet_KeyNotExists()
         $list->set('k4', 'v4', 1);
         $list->set('k5', 'v5', 4);
 
-        $this->assertEquals(null, $list->get('KEY_NOT_EXISTS'));
+        $this->assertNull($list->get('KEY_NOT_EXISTS'));
```

Comment:
it's also my bug, it's better to trigger `OutOfBoundsException`

Reply:
It looks good to throw the ```OutOfBoundsException``` if the key is not existed in that array.

How about this [line](https://github.com/peter279k/php-list/blob/9aebc11bde4ffab2f76bfb1396ff1813eae93230/tests/PriorityMapTest.php#L138)? I think the unknown key will throw the same exception, too.

Reply:
`has ` used to check if key exists, so it's normal that key may not exists, its not excepted situatiod, but if you get key, which not exists, its exception.

Reply:
You're right. The ```has``` method just check whether the key is existed.
I will not modify this and keep the previous approach.

If having any request changes, feel free to send the comment :).

Reply:
i though we deal to trigger OutOfBoundsException when key don't exists